api/client/service/update.go

```diff
@@ -122,8 +117,8 @@ func updateService(spec *swarm.ServiceSpec, flags *pflag.FlagSet) error {
 	updateString(flagName, &spec.Name)
 	updateLabels(flags, &spec.Labels)
 	updateString("image", &cspec.Image)
-	updateSlice("command", &cspec.Command)
-	updateSlice("arg", &cspec.Command)
+	updateStringToSlice(flags, "command", &cspec.Command)
```

Comment:
@aaronlehmann Apologies.

Let's speak in code:

```go
func FromCommandArgs(command, args []string) (entrypoint, cmd []string) {
  if len(command) == 0 {
    // when no command is provided, we just set everything as cmd.
    // subtle: if entrypoint on image is empty, promote args
    return nil, args
  }
  
  // We can do either of the following here
  return command[:1], command[1:] // this is currently implemented
  // Or this
  return command, args // I am not sure about this because entrypoint is confusing.
  // Or this
  return command[:1], append(command[1:], args...)
}
```

However, I think defining these in terms of entrypoint and cmd is just continuing the madness, so let's repeat the definitions:

- `Command` - the full command and arguments.
- `Args` - The arguments to a command, possibly with the command itself.

For use with Docker CLI, only `Args` should be used, unless attempting to override entrypoint.